src/components/EditReviewComment.vue

```diff
@@ -53,12 +53,14 @@ export default {
     this.new_review_comment = this.item.review_comment;
```

Comment:
Didn't notice this when the functionality was introduced, but `new_review_comment` should be camelCase.